sktime/contrib/dictionary_based/PAA.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pandas as pd
+
+from sktime.transformers.base import BaseTransformer
+
+
+class PAA(BaseTransformer):
+
+    def __init__(self,
+                 num_intervals=8,
+                 dim_to_use=0
+                 ):
+        self.num_intervals = num_intervals
+
+        self.dim_to_use = dim_to_use
+
+        self.num_insts = 0
+        self.num_atts = 0
+
+    def fit(self, X, y=None, **kwargs):
+        if isinstance(X, pd.DataFrame):
+            if isinstance(X.iloc[0, self.dim_to_use], pd.Series):
+                X = np.asarray([a.values for a in X.iloc[:, 0]])
+            else:
+                raise TypeError("Input should either be a 2d numpy array, or a pandas dataframe containing "
+                                "Series objects")
+
+        self.num_atts = X.shape[1]
+        self.is_fitted_ = True
+
+        return self
+
+    def transform(self, X):
+        if not self.is_fitted_:
```

Comment:
see sklearn's `check_is_fitted()` function, to keep checks consitent (here and in other files)